### PR TITLE
CAT-1982 Handled Camera and Flash brick correctly

### PR DESCRIPTION
**Pull request by thmq**

### CODE REVIEW

File: catroid/src/org/catrobat/catroid/camera/CameraManager.java [hunk 1]
```diff
@@ -144,6 +144,17 @@ public boolean hasFrontCamera() {
...
+	public boolean hasCurrentCameraFlash () {
+		if ((currentCameraID == backCameraID) && hasFlashBlack()) {
+			return true;
+		}
+
+		if ((currentCameraID == frontCameraID) && hasFlashFront()) {
+			return true;
+		}
+		return false;
+	}
```
Comment (robertpainsi) [inline catroid/src/org/catrobat/catroid/camera/CameraManager.java]: Simplify :wink:

Comment (fwinkelbauer) [inline catroid/src/org/catrobat/catroid/camera/CameraManager.java]:
```
boolean isBackWithFlash = (currentCameraID == backCameraID) && hasFlashBlack();
boolean isFrontWithFlash = (currentCameraID == frontCameraID) && hasFlashFront();

return isBackWithFlash || isFrontWithFlash;
```

oh and it should read hasFlash**Back**() not hasFlash**Black**()

Comment (thmq) [inline catroid/src/org/catrobat/catroid/camera/CameraManager.java]: are you alright with @fwinkelbauer's suggestions?

Comment (robertpainsi) [inline catroid/src/org/catrobat/catroid/camera/CameraManager.java]: :ok_hand:

**Pull request merged**